examples/monitors-signals/main.tf

```diff
@@ -55,3 +55,59 @@ resource "chronosphere_monitor" "monitor_with_signal" {
     }
   }
 }
+
+resource "chronosphere_monitor" "monitor_with_logging_query" {
```

Comment:
would prefer if this was in a new directory, `examples/monitor-logging`, as the goal isn't to test monitor with signals but logging queries